packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -330,6 +339,15 @@
 			@click="saveModelConfiguration"
 		/>
 	</Dialog>
+	<tera-save-asset-modal
+		:initial-name="optimizedInterventionPolicy?.name"
+		:is-visible="showSaveInterventionPolicy"
+		:asset="optimizedInterventionPolicy"
+		:asset-type="AssetType.InterventionPolicy"
+		@close-modal="showSaveInterventionPolicy = false"
+		@on-save="onSaveForReuse"
+		is-overwriting
```

Comment:
I made a change to the overwriting code you may want to check

Reply:
thanks, changed the prop and it should be good now!